lib/streamlit/hashing.py

```diff
@@ -145,39 +149,26 @@ def is_simple(obj):
     return None
 
 
-def _hashing_error_message(start):
+def _hashing_error_message(bad_type):
     return textwrap.dedent(
         """
-        %(start)s
-
-        **More information:** to prevent unexpected behavior, Streamlit tries
-        to detect mutations in cached objects defined in your local files so
-        it can alert you when the cache is used incorrectly. However, something
-        went wrong while performing this check.
+        Cannot hash object of type %(bad_type)s
 
-        This error can occur when your virtual environment lives in the same
-        folder as your project, since that makes it hard for Streamlit to
-        understand which files it should check. If you think that's what caused
-        this, please add the following to `~/.streamlit/config.toml`:
+        As part of caching some code, Streamlit encountered an object of
```

Comment:
We can make this read a little more smoothly -- something like:

* While caching some code...
 
or

* In the course of caching some code...